tun/tun_linux.go

```diff
@@ -464,7 +486,13 @@ func (tun *NativeTun) BatchSize() int {
 
 func (tun *NativeTun) DisableUDPGRO() {
 	tun.writeOpMu.Lock()
-	tun.udpGRO = false
+	tun.gro.disableUDPGRO()
+	tun.writeOpMu.Unlock()
+}
+
+func (tun *NativeTun) DisableTCPGRO() {
```

Comment:
Docs on these two? Guidance for when callers would want to call them?

(aside: is "GRO" even defined anywhere in the godoc, btw? That probably makes these methods look even more mysterious.)

Reply:
added in https://github.com/tailscale/wireguard-go/pull/33/commits/615c928261240e0007566e9f0285a589ea28698c